DirectEmployers/MyJobs
# [PD 3120] match table layout with user management

## What's this PR do?
PR makes the Outreach Record table look similar to the user management table

## Screenshots (if appropriate)
![screen shot 2017-04-07 at 9 33 45 am](https://cloud.githubusercontent.com/assets/22029077/24803558/9eea7984-1b79-11e7-9efa-890eeb0f1072.png)
THIS IS THE USER MANAGEMENT TABLE

![screen shot 2017-04-07 at 9 33 53 am](https://cloud.githubusercontent.com/assets/22029077/24803568/a4cbe40a-1b79-11e7-95ab-f646afd7e427.png)
THIS IS THE OUTREACH RECORD TABLE BEFORE

![screen shot 2017-04-07 at 10 02 39 am](https://cloud.githubusercontent.com/assets/22029077/24803569/a7d8e49a-1b79-11e7-90e3-d159eb9874b7.png)
THIS IS THE OUTREACH RECORD TABLE AFTER

## Where should the reviewer start?
Start inside of the NUO app and go to the Outreach Records page

## How should this be manually tested?
Pull down the repository and go into the NUO app. When you go to view the outreach records you will see the table there. You can go to user management and go to users and see they match now

## Any background context you want to provide?
